### PR TITLE
Added Usage Example for MobileNetV2

### PR DESCRIPTION
**Pull request by akalakheti**

### CODE REVIEW

**Comment by akalakheti** [comment]
@googlebot I signed it!

**Comment by akalakheti** [comment]
@googlebot I signed it!

**Comment by akalakheti** [comment]
Anything else I could do to get this PR approved?

**Comment by akalakheti** [comment]
@dynamicwebpaige  @fchollet  Sir, could you please review my PR?

**Comment by mihaimaruseac** [comment]
Please follow guideliness at https://www.tensorflow.org/community/contribute/docs_ref and make it a doctest

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 1]
```diff
@@ -159,10 +159,10 @@ def MobileNetV2(input_shape=None,
...
       
   Usage Example:
-  >> from tensorflow.keras.applications import MobileNetV2
-  >> IMG_SHAPE = (224, 224, 3)
-  >> #include_top should be set True if we want to use the output layer of model otherwise it should be set False.
-  >> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
+  >>> from tensorflow.keras.applications import MobileNetV2
+  >>> IMG_SHAPE = (224, 224, 3)
+  >>> #include_top should be set True if we want to use the output layer of model otherwise it should be set False.
+  >>> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: Still not testable. There is no assertion, no output to test against

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 2]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
     ValueError: in case of invalid argument for `weights`,
       or invalid input shape or invalid alpha, rows when
       weights='imagenet'
+      
+  Usage Example:
+  >>> from tensorflow.keras.applications import MobileNetV2
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: Space above this line

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 3]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
...
+  >>> IMG_SHAPE = (224, 224, 3)
+  >>> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: `input_shape=IMG_SHAPE` is the pylint style, same for `include_top=True`.

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 4]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
...
+  >>> model
+  <tensorflow.python.keras.engine.training.Model at 0x7f19e6f42d68>
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: `0x7f19e6f42d68`won't be the same on other runs. Did you test the doctest locally?

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 5]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
...
+
+  Args:
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: Arguments are already documented above.

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 6]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
     ValueError: in case of invalid argument for `weights`,
       or invalid input shape or invalid alpha, rows when
       weights='imagenet'
+      
+  Usage Example:
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: Should come before input/output documentation

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 7]
```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
...
+  >>> from tensorflow.keras.applications import MobileNetV2
+  >>> IMG_SHAPE = (224, 224, 3)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: Better to create a small fake image (using `tf.constant`) and run that through the model

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/applications/mobilenet_v2.py (hunk 3 above)
Comment (akalakheti) [reply tensorflow/python/keras/applications/mobilenet_v2.py]: what should be here instead?

File: tensorflow/python/keras/applications/mobilenet_v2.py (hunk 7 above)
Comment (akalakheti) [reply tensorflow/python/keras/applications/mobilenet_v2.py]: If I created a fake image and passed it to model, wouldn't it give a large array as output?

File: tensorflow/python/keras/applications/mobilenet_v2.py (hunk 4 above)
Comment (akalakheti) [reply tensorflow/python/keras/applications/mobilenet_v2.py]: I tested it on colab.

File: tensorflow/python/keras/applications/mobilenet_v2.py (hunk 3 above)
Comment (mihaimaruseac) [reply tensorflow/python/keras/applications/mobilenet_v2.py]: No need for spaces around `=` in function parameters. Make sure you run pylint on the code and fix any errors

File: tensorflow/python/keras/applications/mobilenet_v2.py [hunk 8]
```diff
@@ -160,10 +160,10 @@ def MobileNetV2(input_shape=None,
       
   Usage Example:
   >>> from tensorflow.keras.applications import MobileNetV2
-  >>> IMG_SHAPE = (224, 224, 3)
-  >>> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
+  >>> IMG_SHAPE=(224, 224, 3)
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/applications/mobilenet_v2.py]: http://google.github.io/styleguide/pyguide.html

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@akalakheti Any update on this PR, please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
@akalakheti Any update on this PR and can you please resolve conflicts? Thanks!

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**